src/components/cart/cart.css

```diff
@@ -0,0 +1,194 @@
+.cart-container-section{
+    height: auto;
+    width: 100%;
+    background: url(../../../public/images/shop-header-bg.jpg) no-repeat center center/cover;
+}
+
+.cart-container-heading{
+    font-weight: 600;
+    font-style: normal;
+    color: #a8e8e2;
+    font-size: 4.375rem;
+    line-height: 70px;
```

Comment:
use rems instead of px

Reply:
No i don't want it to change according to size...that's why it's in px